### ISSUE TITLE
Track errors per field

### ISSUE THREAD

**Issue by XiNiHa** [kamilkisiela/graphql-hive#5639]
- Report the full errors, not only the numbers of them
- Process errors' position and associate it with field-level insights

**Comment by dotansimha** [comment]
Thanks @XiNiHa .

@kamilkisiela @n1ru4l If I understand correctly, to achieve this, we first need send the `errrors[].path`, and not just the count (from Hive agent)?

**Comment by kamilkisiela** [comment]
Yes, but the errors won't have any context (no request/session id, sub-calls to subgraphs or other data sources), we will just know they occurred.
